src/PowerShell.Core.Instrumentation/version.rc

```diff
@@ -0,0 +1,15 @@
+//
+//    Copyright (C) Microsoft.  All rights reserved.
+//
+#include <windows.h>
+#include <ntverp.h>
+
+#define VER_FILETYPE    		VFT_DLL
+#define VER_FILESUBTYPE 		VFT2_UNKNOWN
+#define VER_FILEDESCRIPTION_STR     	"DSC"
```

Comment:
Should this be PowerShellCore?

Reply:
Fixed

Reply:
@dantraMSFT please push your fix so that @adityapatwardhan can sign off.